symfony/framework-bundle/5.1/config/packages/framework.yaml

```diff
@@ -1,4 +1,5 @@
 framework:
+    secret: '%env(APP_SECRET)%'
```

Comment:
with this change, the while `config/packages` folder becomes identical to the one in 4.2
we shoud thus put a symlink to the directory in the recipe for 5.1

Reply:
done!